api4/system_test.go

```diff
@@ -718,6 +718,8 @@ func TestRedirectLocation(t *testing.T) {
 	defer th.TearDown()
 	Client := th.Client
 
```

Comment:
Could you please add like below so it will revert change on test completion?
```
	enableLinkPreviews := *th.App.Config().ServiceSettings.EnableLinkPreviews
	defer func() {
		th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableLinkPreviews = enableLinkPreviews })
	}()
```